driver-core/src/main/com/mongodb/internal/connection/OperationContext.java

```diff
@@ -41,6 +44,24 @@ public OperationContext(final RequestContext requestContext, final SessionContex
         this(NEXT_ID.incrementAndGet(), requestContext, sessionContext, timeoutContext, serverApi);
     }
 
+
+    public static OperationContext todoOperationContext() {
+        // TODO (CSOT) should be removed; used at locations that require an OC, but which do not yet have one available
```

Comment:
These TODOs are intentional, and would get merged into the CSOT branch (which we would check for TODOs before merging, as per a comment there).

I left this open, because I think this would be more appropriate to supply during the crypt work.

Reply:
ACK